localnode/docker-compose_mainnet.yml

```diff
@@ -160,7 +160,7 @@ services:
       op-geth-l2-init-tbc:
         condition: "service_completed_successfully"
     volumes:
-      - ./mainnet-genesis.json:/tmp/genesis.json
+      - ./mainnetgenesis.json:/tmp/genesis.json
```

Comment:
Why get rid of the hyphen?  mainnet-genesis.json is more like the naming for testnet so I think we need to stay consistant.

Reply:
laziness on my end; I downloaded the file without a hyphen and just changed it in the compose file 🤦 will update